.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- Pull Request Template -->
+
+## Related Issue
+
+Resolves #issue_number
+
+<!-- If there is no issue number, the PR will not be merged. Therefore, please ensure that the issue number is added -->
+
+## Description
+
+<!-- Write a brief description of the changes made in the PR. Explain the problem being addressed, or any relevant
+information. -->
+
+## Screenshots
+
+<!-- Add screenshots to preview the changes  -->
+
+## Checklist 
```

Comment:
I think all the points dont make much sense. its obvious most of the time to do this. atleast for anyone who isnt contributing for the first time. 

So its better to remove. @J0SAL Please let us know of your thoughts on this.

Reply:
No, the points make good sense.
A lot of beginners/participants just create a PR for the sake of it, this structure will help provide some formality while creating a PR

Reply:
Sure, I will make the required changes.